itest/list_on_test.go

```diff
@@ -327,6 +327,10 @@ var allTestCases = []*lntest.TestCase{
 		Name:     "query routes",
 		TestFunc: testQueryRoutes,
 	},
+	{
+		Name:     "query routes routehints",
```

Comment:
Using a itest here is overkill, it is already tested in: https://github.com/lightningnetwork/lnd/blob/3707b1fb7032d98dc2a8e881d3ae83e61fbc35e3/lnrpc/routerrpc/router_backend_test.go#L78

Reply:
Therefore commit can be dropped.